google-chart.html

```diff
@@ -338,19 +338,43 @@
     // Promise for the DataView with which the chart is drawn.
     _dataView: null,
 
+    // Promise for the drawn chart.
+    _chartDrawn: null,
+
+    // The current selection in the chart.
+    _selection: null,
+
     // The chart type was changed.
     // We need to create a new chart and redraw.
     _typeChanged: function() {
-      this._chart = this._createChart();
+      this._chart = createChart(this.type, this.$.chartdiv)
+          .then(this._setupChart.bind(this));
```

Comment:
promises arent avail in ie 11. This will be a breaking change that  we'll need to note...and bump the major versionn